### ISSUE TITLE
https://milankragujevic.com/vip-drop-i-lekcije-u-odnosu-sa-korisnicima

### ISSUE THREAD

**Issue by tombh** [browsh-org/browsh#75]
This site is the first non-Chinese site I've seen that is unable to hide the text during the `graphics_builder` screenshot phase.

**Comment by canselcik** [comment]
Likely has to do with the Serbian characters.

**Comment by tombh** [comment]
Yes, it's very likely to do with the fact that Browsh's custom fonts don't support this font range, it's the same with a lot of Chinese characters too. The problem is to do with how my little font maker script crashes at high UTF-8 values: https://github.com/browsh-org/browsh/blob/63d55b85154bdb048a23387ad66db75a898c49a9/webext/contrib/font_maker.py#L10

If anyone with experience of creating custom fonts could help, it'd be much appreciated.

**Comment by LeapofAzzam** [comment]
`.py`? I look, browsh uses 2 programming language. That's Python and Go.

**Comment by tombh** [comment]
Perhaps this comment will help: https://github.com/fontforge/fontforge/issues/3009#issuecomment-549176958
